src/material-experimental/mdc-helpers/_focus-indicators.scss

```diff
@@ -1,25 +1,32 @@
 @use 'sass:map';
 @use '../../material/core/style/layout-common';
+@use '../../material/core/style/focus-indicators';
 
 /// Mixin that turns on strong focus indicators.
 ///
 /// @example
 ///   .my-app {
 ///     @include mat-mdc-strong-focus-indicators($config);
 ///   }
+
 @mixin strong-focus-indicators($config: ()) {
+  @include focus-indicators.private-strong-focus-indicators-helper();
+
   // Default focus indicator config.
   $default-config: (
-    border-style: solid,
-    border-width: 3px,
-    border-radius: 4px,
+    border-color: black,
   );
 
   // Merge default config with user config.
   $config: map.merge($default-config, $config);
-  $border-style: map.get($config, border-style);
-  $border-width: map.get($config, border-width);
-  $border-radius: map.get($config, border-radius);
+
+  @if (&) {
```

Comment:
The `if/else` can replaced with a `@include private-strong-focus-indicators-helper($config, 'mat-mdc');` call since the `:root` logic is handled in the `private-strong-focus-indicators-helper` mixin already.